docs/step-by-step/kubeapps-on-tkg/step-2.md

```diff
@@ -106,16 +106,14 @@ authProxy:
   clientSecret: CLIENT-SECRET
   cookieSecret: COOKIE-SECRET
   additionalFlags:
-    - --scope=openid email groups
```

Comment:
I was wondering why this was, but see it's because we switched the other month to pass a value (with the same default).
```
            - --scope={{ .Values.authProxy.scope }}
```

Reply:
Yep, exactly, I'm still trying to remove them from all the guides and personal dev files, this way configuring the auth-proxy becomes simpler